docs/earth-relief.rst

```diff
@@ -70,12 +70,12 @@ to latitudes ±60˚ and are only available over land.  When these grids are acce
 tiles to fill in the missing ocean values. If you just want the original land-only SRTM tiles
 you may use the special names @srtm_relief_03s or @srtm_relief_01s instead. Almost all grids
 are available in both gridline- and pixel-registered formats except the original pixel-registered
-SRTM15+V2.3 (here called @earth_relief_15s) and the gridline-registered SRTM tiles.
+SRTM15+V2.4 (here called @earth_relief_15s) and the gridline-registered SRTM tiles.
 
 Data References
 ~~~~~~~~~~~~~~~
 
-#. SRTM15+V2.3: [https://doi.org/10.1029/2021EA002069].
+#. SRTM15+V2.4: [https://doi.org/10.1029/2019EA000658].
 #. SYNBATH_V1.2: [https://doi.org/10.1029/2021EA002069].
```

Comment:
```suggestion
#. SRTM15+: [https://doi.org/10.1029/2019EA000658].
#. SRTM15+ (V2.3+): [https://doi.org/10.1029/2021EA002069].
#. SYNBATH (V1.2+): [https://doi.org/10.1029/2021EA002069].
```
I think this should be sufficient to prevent us from needing to update this section after each release. What do you think @PaulWessel?

Reply:
I think so. But whether 2.4 is Tozer or Sandwell is indeed not clear until David answers

Reply:
Thanks for checking. I will leave the PR open for now.